sa/model.go

```diff
@@ -567,6 +496,7 @@ func authzPBToModel(authz *corepb.Authorization) (*authz2Model, error) {
 		// convert the integer ID to a string.
 		id, err := strconv.Atoi(*authz.Id)
 		if err != nil {
+			fmt.Println("here")
```

Comment:
:scissors:
```suggestion
```

Reply:
Somewhat surprised this is the only debug print that survived...